### PR TITLE
Curate provisioning

### PR DESCRIPTION
**Pull request by luis261**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I am changing.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

There are 763 public repositories matching this topic

---

### CODE REVIEW

File: topics/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+topic: provisioning
+display_name: Provisioning
+short_description: the process of setting up IT infrastructure.
```
Comment (ahpook) [inline topics/index.md:4]:
```suggestion
short_description: Tools to help with creating and configuring IT infrastructure.
```

**Review by ahpook — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution! I requested one change to make a complete sentence of the short description. If you feel like it, it'd be nice (but not required) to add a `related` key with some adjacent topics.

**Comment by luis261** [comment]
Alright, thank you for the swift response and the suggestion! I also added a `related` key with (:

**Comment by luis261** [comment]
I just realized I messed up the path earlier. Sorry about that, don't know how that happened. Should be good to go now though

**Review by ahpook — APPROVED** [review]
LGTM, thanks for the improvements!

**Pull request merged**